src/game.cpp

```diff
@@ -2616,6 +2619,10 @@ void Game::processKeyboardInput(VolatileRunFlags *flags,
 
 	if (input->wasKeyDown(keycache.key[KeyCache::KEYMAP_ID_DROP])) {
 		dropSelectedItem();
+	// Add WoW-style autorun by toggling continuous forward.
+	} else if (input->wasKeyDown(keycache.key[KeyCache::KEYMAP_ID_AUTORUN])) {
+		bool autorun_setting = g_settings->getBool("continuous_forward");
```

Comment:
There's one in defaultsettings.cpp already.

https://github.com/minetest/minetest/blob/master/src/defaultsettings.cpp#L121